serokell-style.md

```diff
@@ -233,47 +230,47 @@ align by max width among them.
 
 Use singular when naming modules (e.g. use `Data.Map` and
 `Data.ByteString.Internal` instead of `Data.Maps` and
-`Data.ByteString.Internals`). Sometimes it's acceptable to use plural
+`Data.ByteString.Internals`). Sometimes it is acceptable to use plural
 (e. g. `Types`, `Instances`).
 
 ### Export Lists
 
+All modules _must_ have explicit exports.
+
 Format export lists as follows:
 
 ```haskell
 module Data.Set
-       ( -- * The @Set@ type
-         Set
-       , empty
-       , singleton
+  ( -- * The @Set@ type
+   Set
+  , empty
+  , singleton
 
-         -- * Querying
-       , member
-       ) where
+   -- * Querying
+  , member
+  ) where
 ```
 
 Some clarifications:
 
-1. Use 7 spaces indentation for export list (so that bracket is below
-   the first letter in module name).
+1. Use 2 spaces indentation for export list, like in all other cases.
 2. You can split export list into sections or just write all as single section.
-3. It is strongly adviced to sort each section alpabetically. However,
-   classes, data types and type aliases should be written before
+3. It is strongly advised to sort each section alpabetically. However,
+   within each section, classes, data types and type aliases should be written before
    functions.
 
 ### Imports
 
 Imports should be grouped in the following order:
 
-0. Implicit import of custom prelude (for example [`universum`](https://github.com/serokell/universum)) if used.
+0. Implicit import of custom prelude (for example [`universum`](https://github.com/serokell/universum)) if you are using one. You _may_ also use [`base-noprelude`](https://hackage.haskell.org/package/base-noprelude) in order to avoid importing your custom prelude at all.
 1. Everything from hackage packages or from your packages outside current project.
```

Comment:
What is _project_?

Reply:
I suggest loosely defining it as "everything that is in your current repository".

Reply:
Ok, then probably `s/project/repository`?

Reply:
Or maybe keep _current project_ and just write that normally by project we mean current repository, but in some cases it's ok to treat it differently.

Reply:
Aha, ok, didn't notice.